app/components/organisation/school_overview_component.rb

```diff
@@ -0,0 +1,11 @@
+class Organisation::SchoolOverviewComponent < ViewComponent::Base
```

Comment:
As a convention, module names or scopes in general should be plural, so I would call it Organisations::SchoolOverviewComponent. Not sure why you are putting this under `organisation` though

Reply:
1. This is the first in a set of components that are all to do with the organisations dashboard. I envisage them all being under `organisation`
2. There is already a component called `SchoolOverviewComponent